buzzfeed-style-quiz/src/index.html

```diff
@@ -53,4 +51,5 @@ <h2 class="quiz-question">Q6:</h2>
 
 <div class="quiz-result"></div>
 
-
+<!-- Magic things that make the quiz work behind the scenes! -->
+<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js" type="text/javascript"></script>
```

Comment:
We need to include this so when students copy our codepen the appropriate jQuery libraries are linked